extensions/common/context.cc

```diff
@@ -520,5 +534,19 @@ bool getAuditPolicy() {
   return shouldAudit;
 }
 
+bool sanitizeBytes(std::string* buf) {
+  char* start = buf->data();
+  const char* const end = start + buf->length();
+  bool modified = false;
+  while (start < end) {
+    if (flatbuffers::FromUTF8(const_cast<const char**>(&start)) < 0) {
+      *start = ' ';
+      start += 1;
```

Comment:
It seem the start could be advanced if the first byte is legit (e.g. the first byte indicate len = 2  b'11000000').
I *think* your goal is to replace b'1100000' by SPACE and start from the byte after b'11000000'. 

https://github.com/google/flatbuffers/blob/df2df21ec1be2468106fed10c1533eacc1cf0d2e/include/flatbuffers/util.h#L528


my imagined nagative case is a two byte string: b'11000000' b'11000000' 

Can you confirm?

Reply:
Good point. Missed this with the cast. PTAL again, and updated test case with \C0.

Reply:
LGTM!